gemrb/GUIScripts/GUICommonWindows.py

```diff
@@ -2070,3 +2070,12 @@ def RestPress ():
 def RealRestPress ():
 	GemRB.RestParty(0, 0, 1)
 	return
+
+def SwitchPCByKey (wIdx, key, mod):
+	if key >= 49 and key <= 54:
+		GemRB.GameSelectPCSingle(key - 48)
```

Comment:
shouldn't calling `GameSelectPCSingle` ought to call `SelectionChanged` which in turn ought to call `RunSelectionChangeHandler`